orx-jvm/orx-kinect-common/build.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
+    api project(":orx-depth-camera")
```

Comment:
I'd recommend using Kotlin DSL if you're going to be writing a new build script

Reply:
Good point, although surrounding scripts are still `.gradle` I didn't want to accidentally brake something. I will check if migrating to `.gradle.kts` now brings any side effects.

Reply:
I'm pretty sure you can rewrite the second line as `api(project(":orx-depth-camera"))`, then rename the file to `build.gradle.kts` and you'll have an equivalent build script.

Reply:
done